lib/verbose.js

```diff
@@ -0,0 +1,19 @@
+import {debuglog} from 'node:util';
+import process from 'node:process';
+
+export const verboseDefault = debuglog('execa').enabled;
```

Comment:
Using `debuglog()` directly would print `EXECA` followed by the PID, which seems too verbose to me.
So I am only using it to get the `NODE_DEBUG` detection behavior.